java-container/google-cloud-container/src/test/java/com/google/cloud/container/v1/it/ITSystemTest.java

```diff
@@ -180,4 +182,14 @@ public void listNodePoolsTest() {
       }
     }
   }
+
+  private static String getContainerNetworkName() {
+    String name =
+        System.getProperty(CONTAINER_NETWORK_ENV_NAME, System.getenv(CONTAINER_NETWORK_ENV_NAME));
```

Comment:
Do you really need from both? I think one of them suffices.

Reply:
I have seen this [pattern](https://github.com/googleapis/google-cloud-java/blob/main/java-tasks/google-cloud-tasks/src/test/java/com/google/cloud/tasks/v2beta3/CloudTasksSmokeTest.java#L95-L111) around the code, but otherwise: no there's no need for both.

Reducing complexity to support only `getenv` for now.